docs/features/access_log_filter.md

```diff
@@ -0,0 +1,26 @@
+# Access log filter configuration
+
+Using Envoy's [metadata](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/core/base.proto#core-metadata)
+section you can provide additional configuration to the Control Plane.
+Configuration provided in `metadata.access_log_filter` will be used to set up an access log filter for `Envoy`.
+
+## Filter logs by status code
+
+Envoy allows filtering access logs by status code.
+Variable `metadata.access_log_filter.status_code_filter` should contain operator and status code.
+
+Expected format is: `{operator}`:`{status code}`.
+
+Allowed operators are:
```

Comment:
We should add information if operator should be lower/upper case or it is case insensitive

Reply:
added in new commit